src/pages/background/settings.js

```diff
@@ -1,23 +1,12 @@
-import { getSetting, setSettings } from 'common/helpers'
+import { setSettings } from 'common/interface'
 import { setDefaultIcon } from 'common/interface'
 
 export function initColorMode() {
-  const darkMode = window.matchMedia('(prefers-color-scheme: dark)').matches
-  setColorMode(null, { darkMode })
+  const theme = window.matchMedia('(prefers-color-scheme: dark)').matches
+  setColorMode(null, { theme })
 }
```

Comment:
Can we actually delete this `initColorMode()`?

Reply:
Actually, yes! In that case I'm gonna probably move the color theme function to another file. No sense in keeping a file around with a single function in it